ecdsa/src/hazmat.rs

```diff
@@ -236,7 +236,7 @@ pub trait DigestPrimitive: PrimeCurve {
             cmp::Ordering::Equal => field_bytes.copy_from_slice(prehash),
             cmp::Ordering::Less => {
                 // If prehash is smaller than the field size, pad with zeroes
```

Comment:
Can you cite one of the relevant documents here? (possibly FIPS 186-4)

Reply:
I cited FIPS 186-4 in the commit message but FIPS 186-4 lacks the details of ECDSA signature generation algorithm (while [186-5 draft](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.186-5-draft.pdf) and [Suite B Implementors' Guide](https://github.com/briansmith/ring/blob/main/doc/ecdsa.pdf) have).

therefore pushed fc2ab49 to cite RFC and SEC1. WDYT?

Reply:
Looks good, thanks!